### ISSUE TITLE
dotnet-new (new3) framework flag inconsistent

### ISSUE THREAD

**Issue by spboyer** [dotnet/cli#5579]
## Steps to reproduce
`dotnet new console -f netcoreapp1.0` fails 
Same is true for `xunit`, `mstest`

## Expected  behavior
Should be consistent with `web`, `webapi`, `mvc`

## Actual behavior
wants `-F` per `dotnet new console --help`

```
Invalid input switch:
  -f
```

## Environment data
`dotnet --info` output:

```
.NET Command Line Tools (1.0.0-rc4-004734)

Product Information:
 Version:            1.0.0-rc4-004734
 Commit SHA-1 hash:  6990c71e84

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.12
 OS Platform: Darwin
 RID:         osx.10.12-x64
 Base Path:   /usr/local/share/dotnet/sdk/1.0.0-rc4-004734
```

**Comment by livarcocc** [comment]
@mlorbetske

**Comment by mlorbetske** [comment]
This looks like a duplicate of #5571 and should be fixed in the latest build by #5560

cc @seancpeters
